system_tests/logging_.py

```diff
@@ -175,3 +176,25 @@ def test_create_sink_bigquery_dataset(self):
         sink.create()
         self.to_delete.append(sink)
         self.assertTrue(sink.exists())
+
+    def test_create_sink_pubsub_topic(self):
+        from gcloud import pubsub
+
+        # Create the destination topic, and set up the IAM policy to allow
+        # Cloud Logging to write into it.
+        pubsub_client = pubsub.Client()
+        topic = pubsub_client.topic(TOPIC_NAME)
+        topic.create()
+        self.to_delete.append(topic)
+        policy = topic.get_iam_policy()
+        policy.owners.add(policy.group('cloud-logs@google.com'))
+        topic.set_iam_policy(policy)
+
+        TOPIC_URI = 'pubsub.googleapis.com/%s' % (topic.full_name,)
```

Comment:
Go ahead and merge. I don't think this PR needs to be in to be considered "done". The code works just fine. RE: cleverness, we use that resource path every time we make an API request, not sure what makes it clever?